src/ResourceManager/Resources/Commands.Resources/help/Stop-AzureRmDeployment.md

```diff
@@ -0,0 +1,193 @@
+---
+external help file: Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml
+Module Name: AzureRM.Resources
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/stop-azurermdeployment
+schema: 2.0.0
+---
+
+# Stop-AzureRmDeployment
+
+## SYNOPSIS
+Cancal a running deployment
+
+## SYNTAX
+
+### StopByDeploymentName (Default)
+```
+Stop-AzureRmDeployment [-Name] <String> [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### StopByDeploymentId
+```
+Stop-AzureRmDeployment -Id <String> [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### StopByInputObject
+```
+Stop-AzureRmDeployment -InputObject <PSDeployment> [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmDeployment** cmdlet cancels a deployment at subscription scope that has started but not completed.
+To stop a deployment, the deployment must have an incomplete provisioning state, such as Provisioning, and not a completed state, such as Provisioned or Failed.
+
+To create a deployment at subscription scope, use the New-AzureRmDeployment cmdlet.
+
+This cmdlet stops only one running deployment. Use the *Name* parameter to stop a specific deployment.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\>Stop-AzureRmDeployment -Name "deployment01"
+```
+
+This command cancels a running deployment "deployment01" at the current subscription scope.
+
+### Example 2
+```
+PS C:\>Get-AzureRmDeployment -Name "deployment01" | Stop-AzureRmDeployment
+```
+
+This command gets the deployment "deployment01" at the current subscription scope and cancels it. 
+
+## PARAMETERS
+
+### -ApiVersion
+When set, indicates the version of the resource provider API to use.
+If not specified, the API version is automatically determined as the latest available.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Id
+The fully qualified resource Id of the deployment.
+example: /subscriptions/{subId}/providers/Microsoft.Resources/deployments/{deploymentName}
+
+```yaml
+Type: String
+Parameter Sets: StopByDeploymentId
+Aliases: DeploymentId, ResourceId
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+The deployment object.```yaml
```

Comment:
@Tiano2017 same comment about the trailing ` ```yaml ` string

Reply:
fixed.